app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store instances of `Person` by their names. This approach assumes that all names are unique. If there are duplicate names, this will lead to overwriting existing entries. Consider handling potential name collisions or ensuring that names are unique.